README.md

```diff
@@ -147,45 +146,45 @@ plugsconfig.json
 | GET | /raw | - | - | Show Raw Data of the Relay |
 
 ### WebSocket
-You can subscribe to the following events without any authentification:  
-Route `/webuser`, Payload: `{"event": "subscribe_totalpower", "data_payload": {}}` this will push you total power usage.  
-You can subscribe to the applications log with your webtoken as authentication:  
-Router `/webuser`, Payload `{"event": "subscribe_logs", "data_payload": {"webtoken": "Your Admin Webtoken"}}` you can also add the ID parameter to subscribe to a specific pm2_process log.  
+You can subscribe to the following events without authentication:  
+Route `/webuser`, Payload: `{"event": "subscribe_totalpower", "data_payload": {}}` This will give you the total power consumption.  
```

Comment:
push is more accurate because you just need to establish a socket and then do nothing to get this information